.cds/build-manager-ng-app.pip.yml

```diff
@@ -0,0 +1,26 @@
+version: v1.0
+name: build-manager-ng-app
+parameters:
+  name:
+    type: string
+    description: npm module name
```

Comment:
This description doesn't mean nothing right?

Reply:
Yes, we should update it ;)

Reply:
The pipeline requires a parameter which is the name of the module you want to build.

What's your purpose ? :)

Reply:
It could simply reflect the `name`.

```txt
Build OVH Manager AngularJS applications
```

Reply:
It's not the name of the pipeline, it's the name of your npm package that you want to build like @ovh-ux/manager-sms or @ovh-ux/manager-core

-> `npm module name` is the description of the parameter "name"